.gitignore

```diff
@@ -11,6 +11,9 @@ config.js
 *.pdf
 archives
 
+# Cabal config files
+.cabal.yml

```

Comment:
i'm not super sure about yml, but if everyone else feels that's good then let's do it

it's more convenient and prettier though :)

Reply:
Ha! I knew someone would call this out! :D

I'm cool with any format but did yaml just to get it working fast. Plus those yaml lists look so good. Also, Homebase uses yaml for its config. But if json or something else is more conventional or fitting that's cool too. How do you feel, @karissa and @noffle?

Reply:
I'm fine with yaml